tests/unit/test_auto_deploy_task.py

```diff
@@ -193,6 +193,7 @@ async def test_setup_task(
 ):
     assert client.app
     assert auto_deploy_task.TASK_NAME in client.app
+    await asyncio.sleep(1)
     while client.app["state"][auto_deploy_task.TASK_NAME] == State.STARTING:
```

Comment:
for later:
we now make use of tenacity so that the loops are not waiting indefinitely but for a maximum defined amount of time. This usually looks like so:
```python
_TENACITY_RETRY_PARAMS = dict(
    reraise=True,
    retry=retry_if_exception_type(AssertionError),
    stop=stop_after_delay(30),
    wait=wait_fixed(0.1),
)

async for attempt in AsyncRetrying(**_TENACITY_RETRY_PARAMS):
        with attempt:
            print(
                f"setting up task"
            )
            assert client.app["state"][auto_deploy_task.TASK_NAME] == State.RUNNING
            print("... task is finished setting up.")
```

Reply:
thanks!